pkg/crc/oc/oc.go

```diff
@@ -60,15 +60,15 @@ func (oc OcConfig) RunOcCommandPrivate(args ...string) (string, string, error) {
 	return oc.runner.RunPrivate(args...)
 }
 
-func NewOcConfig(runner OcRunner, context string, clusterName string) OcConfig {
+func NewOcConfig(runner ocRunner, context string, clusterName string) OcConfig {
 	return OcConfig{
 		runner:  runner,
 		Context: context,
 		Cluster: clusterName,
 	}
 }
 
-func (oc OcConfig) WaitForOpenshiftResource(resource string) error {
+func WaitForOpenshiftResource(oc OcConfig, resource string) error {
```

Comment:
I would prefer to have this function under a proper type (so that we can test more and mock them). 
The majority of functions using RunOcCommand are statics in the package cluster. Only 3 were out of this package.